tf/tfprocess.py

```diff
@@ -120,6 +120,7 @@ def init_net(self, next_batch):
 
         # Set adaptive learning rate during training
         self.cfg['training']['lr_boundaries'].sort()
+        self.warmup_steps = self.cfg['training'].get('warmup_steps', 0)
         self.lr = self.cfg['training']['lr_values'][0]
```

Comment:
`self.lr = ...` can be removed here, since computing LR is done before the first training step now.